### PR TITLE
Fix Shipping Simulator PropType Error

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?

There was a PropType console error in the Product Page related to the *Shipping Simulator*. I fixed by changing the type from *numeric* to *string*.

#### How should this be manually tested?

[Workspace](https://elvis--storecomponents.myvtex.com/)

- Visit any product in the store
- Check if there is any Proptype errors of `seller`

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ShippingSimulator/index.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ class ShippingSimulator extends Component {
     intl: intlShape.isRequired,
     client: PropTypes.object,
     skuId: PropTypes.string,
-    seller: PropTypes.number,
+    seller: PropTypes.string,
```
Comment (luciannojunior) [inline react/components/ShippingSimulator/index.js]: In the [docs](https://github.com/vtex-apps/store-components/blob/master/react/components/ShippingSimulator/README.md#configuration) it says this prop is required. You could change it here (or there)

(Might be a good time to include the other ones there)

Comment (thayannevls) [reply react/components/ShippingSimulator/index.js]: Done!

File: react/components/ShippingSimulator/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ class ShippingSimulator extends Component {
     intl: intlShape.isRequired,
     client: PropTypes.object,
     skuId: PropTypes.string,
```
Comment (luciannojunior) [inline react/components/ShippingSimulator/index.js]: missing `isRequired` in `skuId`

Comment (thayannevls) [reply react/components/ShippingSimulator/index.js]: Done

**Pull request merged**